src/gui/src/MainWindow.h

```diff
@@ -26,7 +26,7 @@
 #include <QSettings>
 #include <QProcess>
 #include <QThread>
-
+#include <signal.h>
```

Comment:
Shouldn't this be <csignal> in C++ code?

Reply:
Looks good otherwise.

Reply:
Looking into this again, I don't see any calls to things from `signal.h` in this header. What was the compile error? The include should be added to the file where the functionality is used.